templates/articles/_add_comment.html

```diff
@@ -0,0 +1,27 @@
+<turbo-frame id="add_comment">
+{% if user.is_authenticated %}
+    <form action="{% url 'add_comment' article.slug %}" class="card comment-form" method="post">
+    {% csrf_token %}
+        <div class="card-block">
+                <textarea
+                        class="form-control"
+                        placeholder="Write a comment..."
+                        rows="3"
+                        name="body"
+                ></textarea>
+        </div>
+        <div class="card-footer">
+            <img
+                    src="{{ user.profile.img }}"
+                    class="comment-author-img"
+            />
+            <span class="comment-author">Commenting as <a href="">{{ user }}</a></span>
+            <button class="btn btn-sm btn-primary" type="submit">Post Comment</button>
+        </div>
+    </form>
+{% else %}
+    <div class="card m-5">
+        <button onclick="window.location.href='/login';" class="btn btn-sm btn-block btn-primary">Login to Comment</button>
```

Comment:
Would be nice to use an actual link here for accessibility.

Reply:
You can also add a redirect query param so the user get put back here after login.

Reply:
Yea, that was left over from a previous implementation. That doesn't even show up with the current way I have the `turbo-frame` loading it. If a user isn't logged in the the comment box just doesn't load.